### PR TITLE
[Fix] Activity Centre UI bug

### LINKED ISSUE #14231: Activity Center is not closing upon tapping close button
## Problem 

Activity Center is utilising the `popover` view/component to display. The `:hide-popover` event is dispatched upon the user tapping the close button. However, the Activity Center is NOT closing.

### Reproduction

1) Open Status
2) Navigate to Activity Center
3) Try to close Activity Center

### Additional Information

- Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by smohamedjavid**
Fixes #14231

### Summary

Contains fixes for Activity Center UI issues. 

**Design** 

([Figma Link](https://www.figma.com/file/eDfxTa9IoaCMUy5cLTp0ys/Shell-for-Mobile?node-id=3806%3A586138))

#### Areas impacted

- Activity Center (Notifications Center)

**Platforms**

- Android
- iOS

**Steps to test**

1. Open the `Status` application
2. Navigate to the `Communities` or `Messages` tab
3. Open  `Activity Center`

status: ready

### CODE REVIEW

File: src/status_im2/contexts/communities/home/view.cljs [hunk 1]
```diff
@@ -8,6 +8,7 @@
             [quo2.foundations.colors :as colors]
             [quo2.components.community.discover-card :as discover-card]
             [quo2.components.navigation.top-nav :as topnav]
+            [status-im.utils.config :as config]
```
Comment (flexsurfer) [inline src/status_im2/contexts/communities/home/view.cljs]: status-im2.setup.config

File: src/status_im2/contexts/communities/home/view.cljs [hunk 2]
```diff
@@ -109,7 +110,17 @@
                                           colors/neutral-5
                                           colors/neutral-95)}}
       [rn/view {:flex 1}
-       [topnav/top-nav {:type :default}]
+       [topnav/top-nav {:type :default
+                        :open-activity-center #(do
```
Comment (flexsurfer) [inline src/status_im2/contexts/communities/home/view.cljs]: better to move this to a function

**Comment by smohamedjavid** [comment]
@flexsurfer - This PR doesn't require manual testing at the moment as the Activity Centre is not open for testing yet.

File: src/status_im2/contexts/communities/home/view.cljs [hunk 3]
```diff
@@ -8,12 +8,24 @@
...
             [status-im2.common.plus-button.view :as components.plus-button]
             [utils.re-frame :as rf]
             [i18n.i18n :as i18n]
             [quo2.core :as quo]
             [status-im.ui.screens.communities.community-options-bottom-sheet :as home-actions]))
 
+(defn navigate-to-activity-center []
+  (rf/dispatch [:mark-all-activity-center-notifications-as-read])
+  (if config/new-activity-center-enabled?
+    (rf/dispatch [:show-popover {:view                        :activity-center
```
Comment (ilmotta) [inline src/status_im2/contexts/communities/home/view.cljs]: What do you think of making this dispatch to show the activity center reused in `screens.chat.home/home-tooltip-view`? It seems both will always need to be in sync, but with the duplication someone in the future might forget to update one of them.

**Review by ilmotta — APPROVED** [review]

File: src/status_im/activity_center/core.cljs [hunk 4]
```diff
@@ -6,6 +6,18 @@
             [status-im.utils.fx :as fx]
             [taoensso.timbre :as log]))
 
+;;;; Navigation
+
+(fx/defn open-activity-center
+  {:events [:activity-center/open]}
+  [_]
+  (rf/dispatch [:show-popover {:view                        :activity-center
```
Comment (ilmotta) [inline src/status_im/activity_center/core.cljs]: Just a tiny bit of extra space here.

File: src/status_im/ui2/screens/chat/home.cljs [hunk 5]
```diff
@@ -37,7 +36,8 @@
             [status-im.ui2.screens.chat.components.message-home-item.view :refer [messages-home-item]]
             [status-im.ui2.screens.chat.components.contact-item.view :refer [contact-item]]
             [clojure.string :as str]
-            [status-im2.common.plus-button.view :as components.plus-button])
+            [status-im2.common.plus-button.view :as components.plus-button]
+            [status-im2.setup.config :as config])
```
Comment (flexsurfer) [inline src/status_im/ui2/screens/chat/home.cljs:40]: lint should fail here

Comment (smohamedjavid) [reply src/status_im/ui2/screens/chat/home.cljs:40]: But lint didn't throw any errors.

Screenshot 2022-11-17 at 9 55 37 PM

Comment (flexsurfer) [reply src/status_im/ui2/screens/chat/home.cljs:40]: oh i confused the file, sorry

**Pull request merged**